environment.yml

```diff
@@ -38,26 +35,26 @@ dependencies:
   - pip:
       # --------- Deep Learning --------- #
       # Extra index may need to be on first line 
-      - --extra-index-url https://download.pytorch.org/whl/cu113
-      - torch==1.11.*
+      - --extra-index-url https://download.pytorch.org/whl/cu118
```

Comment:
Is there a reason why you don't use conda for all these packages?
I think it is better to use conda as often as possible when you are building a conda environment to prevent issues/mismatches with dependencies

Reply:
I changed the recipe for conda, and tested a quick training run. It seems to work fine :)